frontend/src/features/AdminCategories/adminCategoriesAPI.ts

```diff
@@ -0,0 +1,10 @@
+const axios = require("axios").default;
+
+export const fetchAdminQuiz = async (link: string) => {
+  try {
+    const response = await axios.get(`${link}`);
+    return response.data;
+  } catch (error) {
+    console.error(error);
+  }
+};
```

Comment:
Passing dynamic link for an api is not ideal since its implying that your `fetchAdminQuiz()` can accept all type of links. Unless this is a callback url for redirecting its understandable but then you would still need to make your main url static/fixed to the actual url this method is going to fetch from.

Code logic can be miss leading since im expecting (based on the function name) that it will fetch from the quiz url but url can be anything so it doesnt match its functions purpose. 

Function name is singular `fetchAdminquiz` i'm expecting function to return only 1 quiz but from the url its the list, so you also need to make the function name match with what the function actually does.